modules/swayimg/hm.nix

```
@@ -0,0 +1,46 @@
+{
+  mkTarget,
+  lib,
+  config,
+  ...
+}:
+mkTarget {
+  name = "swayimg";
+  humanName = "Swayimg";
+
+  configElements =
+    { colors, opacity }:
+    with colors.withHashtag;
+    let
+      swayimgOpacity = lib.toHexString (
+        ((builtins.floor (opacity.applications * 100 + 0.5)) * 255) / 100
```

These parentheses should be optional:

```suggestion
        builtins.floor (opacity.applications * 100 + 0.5) * 255 / 100
```